mlir/test/CMakeLists.txt

```diff
@@ -44,6 +44,12 @@ if(NOT MLIR_MIOPEN_DRIVER_RANDOM_DATA_SEED STREQUAL "none")
   set(MLIR_RANDOM_DATA "-rand ${MLIR_MIOPEN_DRIVER_RANDOM_DATA_SEED} ")
 endif()
 
+# Passed to lit.site.cfg.py.in to set the -pv or -pv_with_gpu flag for the mlir E2E tests.
+set(MLIR_POPULATE_VALIDATION "-pv")
+if(MLIR_MIOPEN_DRIVER_GPU_VALIDATION_ENABLED)
```

Comment:
Do we want some sort of canonicalize_booleans call earlier in this file?

Reply:
like what?

Reply:
https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/blob/miopen-dialect/mlir/test/CMakeLists.txt#L7

Reply:
Don't think it is necessary. This flag is only used in the file. We don't need to pass it to python. So it's not necessary to canonicalize.

Reply:
Got it